### ISSUE TITLE
[BUG] irrelevant FutureWarning printed to STDERR during read_csv

### ISSUE THREAD

**Issue by jangorecki** [rapidsai/cudf#5182]
**Describe the bug**
There is a message being printed out to console (STDERR), which is irrelevant to my actions, seems to be an issue caused by internals of cudf.

**Steps/Code to reproduce bug**

```py
>>> x = cu.read_csv(src_grp, header=0, dtype=['str','str','str','int32','int32','int32','int32','int32','float64'])
/home/jan/anaconda3/envs/cudf/lib/python3.6/site-packages/fsspec/implementations/local.py:33: FutureWarning: The default value of auto_mkdir=True has been deprecated and will be changed to auto_mkdir=False by default in a future release.
  FutureWarning,
```

**Expected behavior**
No message

**Environment overview**
 - Environment location: bare-metal
 - Method of cuDF install: conda

**Environment details**

```sh
cudf/print_env.sh
#bash: cudf/print_env.sh: No such file or directory
```

New environment created as of today (2020-05-13)
```
conda create -y --name cudf
conda activate cudf
conda install -y --name cudf -c rapidsai -c nvidia -c conda-forge -c defaults cudf=0.13 python=3.6 cudatoolkit=10.0
conda install -y --name cudf psutil
```

**Additional context**
If you think that csv data is relevant it can be generated using this code: https://github.com/dask/dask/issues/4372#issuecomment-453899270

**Comment by rgsl888prabhu** [comment]
This issues has been resolved in 0.14.

**Comment by jangorecki** [comment]
@rgsl888prabhu Sorry I was not able to verify on nightly before reporting. I am glad to hear it is fixed. Thank you.
